### PR TITLE
clean up links generated with extra // characters

### PR DESCRIPTION
**Pull request by jodygarnett**
I also found some broken links that were not caught earlier.

### CODE REVIEW

File: _posts/2023-11-27-geoserver-2-24-1-released.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ For the complete list see [2.24.1](https://github.com/geoserver/geoserver/releas
 
 Two improvements have been made to the community module for OAuth2 OpenID-Connect authentication:
 
-* [GEOS-11209](https://osgeo-org.atlassian.net//browse/GEOS-11209) Open ID Connect Proof Key of Code Exchange (PKCE)
-* [GEOS-11212](https://osgeo-org.atlassian.net//browse/GEOS-11212) ODIC accessToken verification using only JWKs URI
+* [GEOS-11209](https://osgeo-org.atlassian.net/browse/GEOS-11209) Open ID Connect Proof Key of Code Exchange (PKCE)
+* [GEOS-11212](https://osgeo-org.atlassian.net/browse/GEOS-11212) OIDC accessToken verification using only JWKs URI
```
Comment (jodygarnett) [inline _posts/2023-11-27-geoserver-2-24-1-released.md:62]: This is a small context fix: the correct abrivation is OIDC

File: _posts/2023-12-18-geoserver-2-23-4-released.md [hunk 2]
```diff
@@ -57,8 +57,8 @@ For the complete list see [2.23.4](https://github.com/geoserver/geoserver/releas
 
 Community module development:
 
-* [GEOS-11209](/browse/GEOS-11209) Open ID Connect Proof Key of Code Exchange (PKCE)
-* [GEOS-11212](/browse/GEOS-11212) ODIC accessToken verification using only JWKs URI
+* [GEOS-11209](https://osgeo-org.atlassian.net/browse/GEOS-11209) Open ID Connect Proof Key of Code Exchange (PKCE)
+* [GEOS-11212](https://osgeo-org.atlassian.net/browse/GEOS-11212) ODIC accessToken verification using only JWKs URI
```
Comment (jodygarnett) [inline _posts/2023-12-18-geoserver-2-23-4-released.md:61]: These are the two broken links being fixed.

File: _posts/2023-12-18-geoserver-2-23-4-released.md [hunk 3]
```diff
@@ -57,8 +57,8 @@ For the complete list see [2.23.4](https://github.com/geoserver/geoserver/releas
 
 Community module development:
 
-* [GEOS-11209](/browse/GEOS-11209) Open ID Connect Proof Key of Code Exchange (PKCE)
-* [GEOS-11212](/browse/GEOS-11212) ODIC accessToken verification using only JWKs URI
+* [GEOS-11209](https://osgeo-org.atlassian.net/browse/GEOS-11209) Open ID Connect Proof Key of Code Exchange (PKCE)
+* [GEOS-11212](https://osgeo-org.atlassian.net/browse/GEOS-11212) OIDC accessToken verification using only JWKs URI
```
Comment (jodygarnett) [inline _posts/2023-12-18-geoserver-2-23-4-released.md:61]: These are the two broken links being fixed.

**Comment by jodygarnett** [comment]
@petersmythe can I ask for you review, as we both worked on these pages.

**Comment by jodygarnett** [comment]
@geospatialtechno are you in position to review this for me?

**Comment by petersmythe** [comment]
Sorry, I missed the request, looks perfect to me.

**Pull request merged**